client/src/components/Search.js

```diff
@@ -1,9 +1,9 @@
-import React from "react";
+import React from 'react';
 
 export default function Search() {
-	return (
-		<div>
-			<input type="text" placeholder="Search our site" />
-		</div>
-	)
+  return (
+    <div className="searchbar">
+      <input type="text" className="searchbar--input" placeholder="Search" />
```

Comment:
Why double dash? Also, would this be better as `.searchbar input`?

Reply:
How about a minimalist css library like [Milligram](https://milligram.io/)? It would save us from writing a lot of css ourselves.